lib/internal/Magento/Framework/CompiledInterception/Test/Unit/CompiledInterceptor/CompiledInterceptorTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Framework\CompiledInterception\Test\Unit\CompiledInterceptor;
+
+use Magento\Framework\Code\Generator\Io;
+use Magento\Framework\CompiledInterception\Generator\CompiledInterceptor;
+
+use Magento\Framework\CompiledInterception\Test\Unit\CompiledPluginList\CompiledPluginListTest;
+use \PHPUnit_Framework_MockObject_MockObject as MockObject;
+
+class CompiledInterceptorTest extends \PHPUnit\Framework\TestCase
```

Comment:
these tests are rather Integration than a unit one

Reply:
fixed, splited tests into unit and integration